app/workflows/schemas/section.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "uk.gov.education/rsd/cctc/section",
+  "$id": "/schemas/section",
```

Comment:
I am not sure how this simplification makes this better organised?

Reply:
Added explanation into the commit message as to why this change is happening (tl;dr; these are relative IDs, not explicitly rooted, which makes referencing them from other files hard)